### PR TITLE
camel-velocity integration test

### PR DESCRIPTION
**Pull request by ozgurkisir-qlik**
camel-velocity integration test added

### CODE REVIEW

File: tests/features/camel-velocity/src/main/java/org/apache/karaf/camel/test/CamelVelocityRouteSupplier.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+        name = "karaf-camel-saxon-test",
+        immediate = true,
+        service = CamelRouteSupplier.class
+)
+public class CamelVelocityRouteSupplier extends AbstractCamelSingleFeatureResultMockBasedRouteSupplier {
+
+
+    @Override
+    protected boolean consumerEnabled() {
+        return  false;
```
Comment (essobedo) [inline tests/features/camel-velocity/src/main/java/org/apache/karaf/camel/test/CamelVelocityRouteSupplier.java:37]:
```suggestion
        return false;
```

File: tests/features/camel-velocity/src/test/test/org/apache/karaf/camel/itest/CamelVelocityITest.java [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+
+@CamelKarafTestHint(isBlueprintTest = true)
+@RunWith(PaxExam.class)
+@ExamReactorStrategy(PerClass.class)
+public class CamelVelocityITest extends AbstractCamelSingleFeatureResultMockBasedRouteITest {
+
+    @Override
+    public void configureMock(MockEndpoint mock) {
+        mock.expectedBodiesReceived("World");
```
Comment (essobedo) [inline tests/features/camel-velocity/src/test/test/org/apache/karaf/camel/itest/CamelVelocityITest.java:33]: It should be "Hello, World" so there is something wrong somwhere

Comment (ozgurkisir-qlik) [reply tests/features/camel-velocity/src/test/test/org/apache/karaf/camel/itest/CamelVelocityITest.java]: Done

File: tests/features/camel-velocity/src/main/java/org/apache/karaf/camel/test/CamelVelocityRouteSupplier.java (hunk 1 above)
Comment (essobedo) [inline tests/features/camel-velocity/src/main/java/org/apache/karaf/camel/test/CamelVelocityRouteSupplier.java:37]:
```suggestion
        return false;
```

File: tests/features/camel-velocity/src/main/java/org/apache/karaf/camel/test/CamelVelocityRouteSupplier.java [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    protected boolean consumerEnabled() {
+        return false;
+    }
+
+    @Override
+    protected void configureProducer(RouteBuilder builder, RouteDefinition producerRoute) {
+        producerRoute.log("calling velocity endpoint")
+                .setHeader("greeting", constant("Hello"))
+                .setBody(builder.constant("World"))
+                .to("velocity:file://%s/classes/greeting.vm".formatted(System.getProperty("project.target")))
```
Comment (essobedo) [inline tests/features/camel-velocity/src/main/java/org/apache/karaf/camel/test/CamelVelocityRouteSupplier.java:45]:
```suggestion
                .toF("velocity:file://%s/classes/greeting.vm, System.getProperty("project.target"))
```
Please use toF instead

**Review by essobedo — APPROVED** [review]

File: tests/features/camel-velocity/src/main/java/org/apache/karaf/camel/test/CamelVelocityRouteSupplier.java [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+    protected boolean consumerEnabled() {
+        return false;
+    }
+
+    @Override
+    protected void configureProducer(RouteBuilder builder, RouteDefinition producerRoute) {
+        producerRoute.log("calling velocity endpoint")
+                .setHeader("greeting", constant("Hello"))
+                .setBody(builder.constant("World"))
+                .toF("velocity:file://%s/classes/greeting.vm, System.getProperty("project.target"))
```
Comment (f2par0) [inline tests/features/camel-velocity/src/main/java/org/apache/karaf/camel/test/CamelVelocityRouteSupplier.java:45]:
```suggestion
                .toF("velocity:file://%s/classes/greeting.vm", System.getProperty("project.target"))
```

**Pull request merged**